update/changelog.go

```diff
@@ -0,0 +1,72 @@
+package update
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"gopkg.in/errgo.v1"
+
+	githubClient "github.com/google/go-github/v45/github"
+
+	"github.com/Scalingo/cli/services/github"
+)
+
+func printRelease(release githubClient.RepositoryRelease) {
+	fmt.Printf("Changelog of the version %v\n\n", release.GetTagName())
+	fmt.Printf("%v\n\n", strings.ReplaceAll(*release.Body, "\\r\\n", "\r\n"))
+}
+
+func ShowLastChangelog() error {
+	ctx := context.Background()
+
+	client := github.New()
+
+	cliLastRelease, err := client.GetLatestRelease(ctx)
+	if err != nil {
+		return errgo.Notef(err, "fail to get last CLI release")
+	}
+	printRelease(*cliLastRelease)
+
+	return nil
+}
+
+func ShowChangelog(cacheVersion, newVersion string) error {
+	cliLastReleases, err := getLastReleases()
+	if err != nil {
+		return errgo.Notef(err, "fail to get last CLI release")
+	}
+
+	beginToSowRelease := false
+	// Read it from the oldest to the most recent.
+	for i := len(cliLastReleases) - 1; i >= 0; i-- {
+		release := cliLastReleases[i]
+		if beginToSowRelease {
+			printRelease(*release)
+		}
+
+		// Detect the release from the cache, we can show changelog from next release.
+		if cacheVersion == release.GetTagName() {
+			beginToSowRelease = true
+		}
+	}
+
+	return nil
+}
+
+// getLastReleases get last 10 releases.
```

Comment:
Is there a reason to limit to 10 reasons? It would be interesting to explain this limit.